webcommon/javascript2.editor/nbproject/org-netbeans-modules-javascript2-editor.sig

```diff
@@ -122,6 +122,11 @@ CLSS public abstract interface static !annotation org.netbeans.modules.javascrip
 intf java.lang.annotation.Annotation
 meth public abstract !hasdefault int priority()
 
+CLSS public abstract interface org.netbeans.modules.javascript2.editor.spi.CompletionProviderEx
```

Comment:
CCing @sdedic

Reply:
Yes, OK, friend API exported to js2.html integration module.